HTML5-frontend/ndt-library.js

```diff
@@ -0,0 +1,234 @@
+/**
+ * @fileoverview This javascript code includes an NDT object that contains the
+ * methods that a customized front end implemetation of the NDT test could
+ * utilize.  The NDT object is self invoked when the library is included on a
+ * page, however, the frontend can customize certain properties when the
+ * interaction is initialized with the startTest method.
+ *
+ * Dependencies: ndt-wrapper.js, ndt-browser-client.js
+ */
+
+
+/**
+ * Immediately invoked function expression that creates NDT library object
+ */
+(function (window) {
+  'use strict';
+
+  /** @constructor */
+  function NDTLibjs() {
+
+    // Look up a suitable NDT server using mlab-ns. The ndtServer property is
+    //  used by the backend javascript to determine which server to test
+    //  against.
+    this.getServer();
+
+    // Initializes simulate property to false.  Can be changed upon starting a
+    // test in the testServer method
+    this.simulate = false;
```

Comment:
Since these are private instance variables do we denote them in any decorator or a prepended underscore?